src/components/global/ContactFormModal.jsx

```diff
@@ -15,27 +16,32 @@ import emailjs from '@emailjs/browser';
 import { PropTypes } from 'prop-types';
 
 const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) => {
+  const [messageSuccess, setMessageSuccess] = useState(false);
+  const [messageFailure, setMessageFailure] = useState(false);
   const form = useRef();
 
   const handleClose = () => {
+    setMessageSuccess(false);
+    setMessageFailure(false);
     setShowContactFormModal(false);
   };
 
   const handleSubmit = (e) => {
     e.preventDefault();
 
     emailjs.sendForm('service_co2agxf', 'template_20pnwni', form.current, 'AkBl59Ya3226OfPyQ').then(
-      (result) => {
-        console.log(result.text);
+      () => {
+        setMessageSuccess(true);
+        e.target.reset();
+        setTimeout(() => {
+          setMessageSuccess(false);
+          setShowContactFormModal(false);
+        }, 5000);
```

Comment:
timeout to close is too long. I thought I had to close it manually before the timer went off, change to 1000